src/behavior.ts

```diff
@@ -24,147 +24,254 @@
  *  THE SOFTWARE.
  */
 
-// d3
-// import Selection = d3.Selection;
-import { Selection, AsterPlotData } from "./dataInterfaces";
-// powerbi.extensibility.utils.interactivity
-import { interactivityBaseService, interactivitySelectionService } from "powerbi-visuals-utils-interactivityutils";
-
+import powerbi from "powerbi-visuals-api";
+import { Selection, AsterDataPoint } from "./dataInterfaces";
+import * as asterPlotUtils from "./utils";
+import ISelectionId = powerbi.visuals.ISelectionId;
+import ISelectionManager = powerbi.extensibility.ISelectionManager;
+import { PieArcDatum } from "d3-shape";
```

Comment:
lets keep d3 beginning naming for d3 methods. 
`import { PieArcDatum as d3PieArcDatum } from "d3-shape";`